backend/src/auth/auth.service.ts

```diff
@@ -12,16 +11,17 @@ export class AuthService {
   ) {}
 
   async login(userDto: LoginUserDto) {
-    const userList = await this.usersService.findAll({
-      username: userDto.username,
-    });
-    if (userList.data.length !== 1) {
-      throw new EntityDoesNotExistError(`User: ${userDto.username}`);
+    try {
```

Comment:
The following suggestion is an enhancement, feel free to ignore it. I think it will be better not to return response JSON in a service, as my understanding is that a service is supposed to be usable application-wide. A controller exposes functionality from services to the outside world.
```suggestion
```